docs/examples/llm/vllm.ipynb

```diff
@@ -0,0 +1,499 @@
+{
```

Comment:
<p>could you also add <code>acomplete</code> and <code>astream_complete</code> examples here to tell user how to use both async functions </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>